modules/ding_nodelist/js/node_blocks.js

```diff
@@ -50,13 +50,29 @@
         $(this).find('.field-type-text-long').toggleClass('element-hidden', true);
       });
 
+      // Added function to set height for title and lead text for news CT.
       $('.node-ding-news.nb-item', context).mouseenter(function() {
         var title_and_lead_height;
-        // Set height for title and lead text.
         title_and_lead_height = $(this).find('.title').outerHeight(true) + $(this).find('.field-name-field-ding-news-lead').outerHeight(true) + 50;
 
         $(this).find('.title-and-lead').css('min-height', title_and_lead_height);
       });
+
+      // Added function to set height for title and lead text for page CT.
+       $('.node-ding-page.nb-item', context).mouseenter(function() {
```

Comment:
Why do it foreach CT and not just for all using `.nb-item` and find the lead somehow programatically?

Reply:
I will changed to be more compact, but I can't add foreach CTs because the events CT use the style and js from basic theme. If I will add foreach CTS the event will not be working. I already tried it.